api/src/client/ApiFormationClient.test.ts

```diff
@@ -1606,4 +1601,35 @@ describe("ApiFormationClient", () => {
       });
     });
   });
+
+  describe("health check", () => {
+    it("returns a passing health check if data can be retrieved sucessfully", async () => {
+      mockAxios.post.mockResolvedValue({});
+      expect(await client.health()).toEqual({ success: true, data: { message: "OK" } });
+    });
+
+    it("returns a failing health check if unexpected data is retrieved", async () => {
+      mockAxios.post.mockRejectedValue({ response: { status: 404 }, message: "" });
+      expect(await client.health()).toEqual({
+        success: false,
+        error: {
+          message: "Bad Gateway",
+          serverResponseBody: "",
+          serverResponseCode: 404,
+          timeout: false,
+        },
+      });
+    });
+
+    it("returns a failing health check if axios request times out", async () => {
+      mockAxios.post.mockRejectedValue({});
+      expect(await client.health()).toEqual({
+        success: false,
+        error: {
+          message: "Gateway Timeout",
+          timeout: true,
+        },
+      });
+    });
+  });
```

Comment:
[sand] The behavior we want to capture is that there's a notification thrown if anything other than 200 is thrown. I don't know what all values we could expect from the API, but it might be nice to have a wildcard test in here.

Reply:
gotcha, checking to see if the response.data.success is true now, if return success false